assets/dns/service.yaml

```diff
@@ -1,5 +1,12 @@
 kind: Service
 apiVersion: v1
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: metrics-tls
+  labels:
+    name: dns
+  name: metrics
+  namespace: openshift-dns
 # name, namespace and labels are set at runtime
```

Comment:
`desiredDNSDaemonSet` still overrides labels, name, and namespace, so it does not make sense to add them to the yaml file.

Reply:
fixed.